src/modules/AddOnProduct.rb

```diff
@@ -2235,7 +2235,7 @@ def add_product_from_cd(url, pth, priority, prodname, check_name)
         end
 
         # ask for a different medium
-        current_url = AskForCD(current_url, prodname)
+        current_url = AskForCD(url, prodname)
```

Comment:
or does it get broken when assigning current_url there in next iteration of loop?